_includes/projects_section.html

```diff
@@ -0,0 +1,85 @@
+<div class="flex flex-col justify-center gap-3 py-12 mt-4">
+  <h2
+    class="text-5xl font-bold text-[{{ site.text-colors.darkblue }}] text-center mb-8"
+  >
+    Projects
+  </h2>
+
+  <div
+    class="w-[80vw] mx-auto bg-[{{ site.bg-colors.gray }}] p-6 rounded-2xl border border-[{{ site.bg-colors.skyBlue }}] space-y-6"
+  >
+    {% for project in site.projects limit: 3 %}
+    <!-- Project Card -->
+    <div
+      class="flex flex-col md:flex-row justify-start md:justify-between items-start space-y-6 md:space-y-0"
+    >
+      <!-- Left Section: Icon and Title -->
+      <div class="flex items-center space-x-3">
+        <div class="w-12 h-12 flex justify-center items-center">
+          <iconify-icon icon="mdi:web" class="text-3xl"></iconify-icon>
+        </div>
+        <h2 class="text-2xl font-bold text-[{{ site.text-colors.base-black }}]">
+          {{ project.title }}
+        </h2>
+      </div>
+
+      <!-- Right Section: Status Tags -->
+      <div class="flex flex-wrap gap-2">
+        {% for status in project.status %}
+        <span
+          class="bg-green-200 text-green-700 px-4 py-2 rounded-full text-sm font-bold"
+          >{{ status }}</span
+        >
+        {% endfor %}
+      </div>
+    </div>
+
+    <!-- Tags Section -->
+    <div class="flex flex-wrap gap-2">
+      {% for tag in project.tags %}
+      <span
+        class="bg-[{{ site.bg-colors.softBlue }}] text-[{{ site.text-colors.blue }}] px-3 py-1 rounded-full text-sm font-bold"
```

Comment:
I suggest not to use two different types of blue color here. The same color with different intensity or opacity will do.

Reply:
the colors are seperate because one is for background and for text seperate

Reply:
Yes, you're absolutely correct, so I'm suggesting a different intensity. I liked how you used `bg-green-200 text-green-700` which is the same color but a different shade and also here `bg-orange-200 text-orange-700`. 

Correct me if I am wrong here, @rpandox .

Reply:
Noted. You are right

Get back to you with the changes

Reply:
Noted. You are right

Get back to you with the changes